packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -112,8 +145,29 @@ const DESCRIPTIONS = {
   queryKeys:
     'Indicates whether a new `keyFetchToken` is required, default to `false`.',
   payload: 'Opaque payload to be forwarded to the device.',
+  paymentMethodId: 'A unique identifier for the payment method.',
+  paymentProvider:
+    'The third-party that assists in the electronic payments (e.g. PayPal, Stripe).',
+  paymentType:
+    'The type of the payment method (e.g., `credit`, `debit`, `prepaid`, or `unknown`).',
+  paypalPaymentError: 'The payment error from PayPal encountered.',
+  planId:
+    'A unique identifier for the [plan](https://stripe.com/docs/api/plans/object).',
+  planMetadata:
+    'Set of key-value pairs used to store additional information about the plan.',
+  planName: 'The name of the plan.',
   ppidSeed:
     'Seed used in `sub` claim generation of JWT access tokens/ID tokens for clients with [Pseudonymous Pairwise Identifiers (PPID)](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/oauth/pairwise-pseudonymous-identifiers.md) enabled. Used to forcibly rotate the `sub` claim. Must be an integer in the range 0-1024. If not specified, it will default to `0`.',
+  previousProduct:
+    'The value of the previous product that the user had been subscribed to.',
```

Comment:
If we look at where this is used in the code, it looks to be the previous product name.

Aside: Also I think this is only used for the support panel route, which IIUC is deprecated: https://mozilla-hub.atlassian.net/browse/FXA-5543, since the Support team are now using the FxA Admin Panel instead.